fastlane/Fastfile

```diff
@@ -2,8 +2,8 @@ fastlane_require "base64"
 fastlane_require "fileutils"
 fastlane_require "json"
 
-IOS_APP_VERSION = "1.6.9"
-ANDROID_APP_VERSION = "1.6.5"
+IOS_APP_VERSION = "1.7.0"
+ANDROID_APP_VERSION = "1.6.6"
```

Comment:
Any reason we don't want these version numbers synchronized?

Reply:
Not a strong reason, but the Android version is a lot simpler, given that it doesn't use the webview for the sites. So, the discrepancy is due to having more iterations for iOS with fixes and tweaks to the webview.